### ISSUE TITLE
feature request - remove "notes" menu item

### ISSUE THREAD

**Issue by GretaFrank** [hzrd149/nostrudel#48]
the "notes" menu item does exactly the same thing as just clicking on noStrudel logo and is redundant

to make things worse "notes" reads almost the same as "notifications" which is the menu item directly next to it

when i try to click "notifications" i often click notes because my brain only processes the first few letters before clicking ...

just remove that pointless menu item and problem solved.

or come up with names that actually read / look / sound different.  for example instead of "notes" have "feed" or "timeline"

but then you will still have the problem because when you want to read your own notes you will have to call that something as well.

better to just remove that useless and redundant menu item.

**Comment by hzrd149** [comment]
I dont think it make sense to remove it, since most users wont know the main noStrudel logo goes to the same place. but I agree it is a little confusing next to notifications.
initially I had it called "home" but that didn't feel right since it didn't explain what was there. changing it to "feed" could make sense though
